pkg/jx/cmd/step_create_task.go

```diff
@@ -352,7 +352,7 @@ func (o *StepCreateTaskOptions) generatePipeline(languageName string, pipelineCo
 
 			var volumes []corev1.Volume
 			for i, step := range task.Spec.Steps {
-				volumes = append(volumes, o.modifyVolumes(&step, task.Spec.Volumes)...)
+				volumes = o.modifyVolumes(&step, task.Spec.Volumes)
```

Comment:
This is the only one that seems to matter for now, but seemed like a good idea to undupe the code in pipeline.go as well.